app/models/request.rb

```diff
@@ -0,0 +1,12 @@
+class Request
+  include DataMapper::Resource
+  
+  property :id, Serial
+  property :date_requested, Date
+  property :owner_id, Integer
+  property :status, String, :default => "not confirmed"
```

Comment:
This could be simpler. A boolean data type would make more sense here. No words in the DB unless they're vital to the running of the application.